tests/test_packages/test_contracts/conftest.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
```

Comment:
does that work having a second `conftest.py` file?

Reply:
it kind of works but ci generates an error. But as we discussed I ll remove this since it is contract related test.